Fitness/Controllers/ClassListViewController.swift

```diff
@@ -351,6 +362,12 @@ extension ClassListViewController: FilterDelegate {
             if !params.instructorNames.isEmpty {
                 guard params.instructorNames.contains(currClass.instructor) else { return false }
             }
+            
+            if !params.tags.isEmpty {
+                guard (currClass.tags.contains { tag in
```

Comment:
you should extend the `Tags` Struct. then you can do `currClass.tags.contains(tag)` and it works.

Here is a great guide! https://useyourloaf.com/blog/swift-equatable-and-comparable/

Reply:
I see what you mean, but here I'm trying to test if the tag array contains any of the elements of another array, so the usual `Array.contains(element:)` can't be used so I thought this would be the simplest way to achieve that.